### PR TITLE
Add fuse support for batchnorm with affine=False

### PR DESCRIPTION
**Pull request by sstsai-adl**
Summary: When batchnorm affine is set to false, weight and bias is set to None, which is not supported in this case. Added a fix to set weights to 1 and bias to 0 if they are not set.

Test Plan: Add unit test for testing fusing conv, batchnorm where batchnorm is in affine=False mode.

Differential Revision: D23977080

### CODE REVIEW

**Review by z-a-f — APPROVED** [review]
LGTM! Thank you!

File: torch/nn/utils/fusion.py [hunk 1]
```diff
@@ -16,6 +16,10 @@ def fuse_conv_bn_eval(conv, bn):
 def fuse_conv_bn_weights(conv_w, conv_b, bn_rm, bn_rv, bn_eps, bn_w, bn_b):
     if conv_b is None:
         conv_b = bn_rm.new_zeros(bn_rm.shape)
+    if bn_w is None:
+        bn_w = torch.ones(bn_rm.shape)
```
Comment (vadimkantorov) [inline torch/nn/utils/fusion.py:20]: should it instead be `torch.ones_like(bn_rm)` to preserve device? or at least `torch.ones(bn_rm.shape, device = bn_rm.device)`?

File: torch/nn/utils/fusion.py [hunk 2]
```diff
@@ -16,6 +16,10 @@ def fuse_conv_bn_eval(conv, bn):
 def fuse_conv_bn_weights(conv_w, conv_b, bn_rm, bn_rv, bn_eps, bn_w, bn_b):
     if conv_b is None:
         conv_b = bn_rm.new_zeros(bn_rm.shape)
```
Comment (vadimkantorov) [inline torch/nn/utils/fusion.py:18]: Should it not be `conv_b = torch.zeros_like(bn_rm)` here as well?

Comment (sstsai-adl) [reply torch/nn/utils/fusion.py:18]: Will create another PR for that refactor.
